modules/porous_flow/include/actions/PorousFlowActionBase.h

```diff
@@ -60,13 +60,13 @@ class PorousFlowActionBase : public Action, public PorousFlowDependencies
   const RealVectorValue _gravity;
 
   /// Name of the mass-fraction variables (if any)
-  const std::vector<VariableName> & _mass_fraction_vars;
+  const std::vector<VariableName> _mass_fraction_vars;
```

Comment:
you can undo that once you have the default

Reply:
see the comment above